Life-Expectancy-and-GDP-Starter/life_expectancy_gdp.ipynb

```diff
@@ -124,12 +135,25 @@
     "print('DESCRIPTION ABOUT THE DATA:')\n",
     "print('===========================')\n",
     "print(gdp_data.describe(include='all'), \"\\n\")\n",
-    "#reduce life expectancy variable name\n",
     "\n",
+    "# Analyzing unique values in Country and Year variables\n",
+    "print('INFORMATION ABOUT THE COUNTRY AND YEARS:')\n",
+    "print('=================================================================')\n",
+    "print(f'Unique Value of Country: {gdp_data.Country.unique()}')\n",
+    "print(f'Unique Value of Year: {gdp_data.Year.unique()}', \"\\n\")\n",
+    "\n",
+    "# Reduce the large name of United States of America.\n",
+    "gdp_data.loc[gdp_data['Country'] == 'United States of America', 'Country'] = 'U.S.A'\n",
+    "\n",
+    "#reduce life expectancy variable name\n",
     "gdp_data.rename(columns={'Life expectancy at birth (years)': 'Life'}, inplace=True)\n",
-    "print('DATAFRAME RENAMED:')\n",
-    "print('===========================')\n",
-    "print(gdp_data.head())"
+    "\n",
+    "# Escale of variable GDP in Billions\n",
+    "gdp_data['GDP'] = (gdp_data['GDP'] / 1e9).round(2)\n",
+    "\n",
+    "print('DATAFRAME RENAMED AND TRANSFORMED:')\n",
+    "print('==================================')\n",
+    "print(gdp_data.sample(10))\n"
```

Comment:
Why are we taking a sample of the dataset to print?

Reply:
I think it's better to use `.sample()` because it gives you random rows from the dataframe which gives us a better overall idea of the df.  In our case, the use of `.head()` only show the rows corresponding to Chile due to the ordering.

Reply:
While they serve almost the same purpose,  I think it's better to use head() since it will also give precisely the info that the df is ordered. the sampling method will be always random and always show something different. We can use the tail() also. in any case, we need to maintain the same "style" in all the code so we should decide if we use head or sample.

Reply:
I agree with Gustavo. Using .head() makes the most sense to showcase the structure of the dataset since we see the top of the dataframe. The overall idea of the df comes from when we're visualizing the data from our graphs. Taking a sample of 10 from the data won't give as much information as we need to have a better overall idea of the df.

The point for seeing the .head() is not to have an overall idea of the dataframe but to just see the data and what type of data we should expect. When combined with .dtypes we can see what types of data is our data. 

Overall, .sample(10) and .head(10) will effectively do the same thing, but to keep with consistency, I think we should use .head(). 

I noticed that this dataframe was printed after some transformations were made. If we want to visually show these changes, then we should not show a random sample, but a specific subset of the data we transformed. For example, find the rows in which United States was changed to USA.